packages/eslint-plugin/docs/rules/prefer-optional-chain.md

```diff
@@ -235,7 +235,7 @@ When this option is `true` the rule will skip operands that are not typed with `
 
 #### ❌ Incorrect for `requireNullish: true`
 
-```ts option='{ "requireNullish": true }'
+```ts option='{ "requireNullish": true }' skipValidation
 declare const thing1: string | null;
 thing1 && thing1.toString();
 ```
```

Comment:
This is a bug in the rule: https://github.com/typescript-eslint/typescript-eslint/issues/8487. For now I'll just add `skipValidation`...